docs/Ariba-batchsource.md

```diff
@@ -23,8 +23,10 @@ annotating metadata, etc.
 **Realm (M)**: Realm name from which the data is to be extracted, for example, S4VALL.   
 **View Template Name (M)**: Name of the view template from which data is to be extracted, for example, RequestFactSystemView.
 
```

Comment:
This description has an example with a link, but it's not a real URL. Remove the link:
API Endpoint (M): Base path of the Ariba API, for example, https://sandbox.api.sap.com/.